modeling.py

```diff
@@ -0,0 +1,146 @@
+import nimfa
+import pandas as pd
+from nfldb_tables import NFLFrames
+from scipy.sparse import csr_matrix as csrm
+
+
+class PositionNMF(object):
```

Comment:
I just pushed what I have to this branch. I'd say it's like...88% of the way there. I have the ideal implementation outlined in the main block of validation.py. Only calculating the actual rmse's left. But I'm about to go climbing. Hopefully I've abstracted away that function enough that it should work in many situations. You're welcome to finish it if you want.